Tests/ColorTests.swift

```diff
@@ -10,6 +10,44 @@ import XCTest
 import ACKategories
 
 final class ColorTests: XCTestCase {
+    
+    func testColorToHex() {
+        var hexColor: String?
+        
+        let clearColor = UIColor.clear
+        hexColor = clearColor.hexString
+        XCTAssertEqual("#000000", hexColor)
+        
+        let blackColor = UIColor.black
+        hexColor = blackColor.hexString
+        XCTAssertEqual("#000000", hexColor)
+        
+        let whiteColor = UIColor.white
+        hexColor = whiteColor.hexString
+        XCTAssertEqual("#FFFFFF", hexColor)
+        
+        let greenColor = UIColor.green
+        hexColor = greenColor.hexString
+        XCTAssertEqual("#00FF00", hexColor)
+        
+        let p3redColor = UIColor(displayP3Red: 0, green: 0, blue: 0, alpha: 1)
+        hexColor = p3redColor.hexString
+        XCTAssertEqual("#000000", hexColor)
+        
+        let image = UIColor.green.image()
+        let patternImageColor = UIColor(patternImage: image)
+        hexColor = patternImageColor.hexString
+        XCTAssertEqual(nil, hexColor)
+        
+        let white = UIColor(white: 1, alpha: 1)
+        hexColor = white.hexString
+        XCTAssertEqual("#FFFFFF", hexColor)
+
+        let rgb = UIColor(red: 0, green: 1, blue: 0, alpha: 1)
```

Comment:
I would split the tests into smaller parts. Each "block" should be IMO a seperate test

Reply:
Done :). Travis build failed cause one of the inits is avaliable from iOS 10, so i added also a check there.